spel/scripts/amigen7-build.sh

```diff
@@ -4,33 +4,68 @@
 # task.
 #
 ##############################################################################
-AMIGENSOURCE="${SPEL_AMIGENSOURCE:-https://github.com/plus3it/AMIgen7.git}"
+PROGNAME="$(basename "$0")"
 AMIGENBRANCH="${SPEL_AMIGENBRANCH:-master}"
 AMIGENMANFST="${SPEL_AMIGENMANFST}"
 AMIGENPKGGRP="${SPEL_AMIGENPKGGRP:-core}"
+AMIGENSOURCE="${SPEL_AMIGENSOURCE:-https://github.com/plus3it/AMIgen7.git}"
 AMIGENSTORLAY="${SPEL_AMIGENSTORLAY:-/:rootVol:4,swap:swapVol:2,/home:homeVol:1,/var:varVol:2,/var/log:logVol:2,/var/log/audit:auditVol:100%FREE}"
+AMIUTILS="/tmp/ami-utils"
 AMIUTILSSOURCE="${SPEL_AMIUTILSSOURCE:-https://github.com/ferricoxide/Lx-GetAMI-Utils.git}"
 AWSCLIV1SOURCE="${SPEL_AWSCLIV1SOURCE:-https://s3.amazonaws.com/aws-cli/awscli-bundle.zip}"
 AWSCLIV2SOURCE="${SPEL_AWSCLIV2SOURCE}"
 BOOTLABEL="${SPEL_BOOTLABEL:-/boot}"
 BUILDNAME="${SPEL_BUILDNAME}"
 CHROOT="${SPEL_CHROOT:-/mnt/ec2-root}"
 CLOUDPROVIDER="${SPEL_CLOUDPROVIDER:-aws}"
-CUSTOMREPORPM="${SPEL_CUSTOMREPORPM}"
 CUSTOMREPONAME="${SPEL_CUSTOMREPONAME}"
+CUSTOMREPORPM="${SPEL_CUSTOMREPORPM}"
 DEVNODE="${SPEL_DEVNODE:-/dev/nvme0n1}"
+ELBUILD="/tmp/el-build"
 EPELRELEASE="${SPEL_EPELRELEASE:-https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm}"
 EPELREPO="${SPEL_EPELREPO:-epel}"
 EXTRARPMS="${SPEL_EXTRARPMS}"
 FIPSDISABLE="${SPEL_FIPSDISABLE}"
+PYTHON3_BIN="/usr/bin/python3.6"
+PYTHON3_LINK="/usr/bin/python3"
 VGNAME="${SPEL_VGNAME:-VolGroup00}"
 
 read -r -a BUILDDEPS <<< "${SPEL_BUILDDEPS:-lvm2 parted yum-utils unzip git}"
 
-PYTHON3_BIN="/usr/bin/python3.6"
```

Comment:
would prefer to keep these separate because they are static globals, instead of user inputs read from ENVs...

Reply:
Moved variables to keep the static globals separate.